microsetta_admin/_api.py

```diff
@@ -39,6 +39,11 @@ def _check_response(response):
         if response.status_code == 401:
             # redirect to home page for login
             output = redirect("/")
+        elif response.status_code > 400:
```

Comment:
What about status code 400 (Bad Request)?

Reply:
Ooo, thank you!  Changed to >= :)